domains/logistics/configure-organization/dataset.ttl

```diff
@@ -0,0 +1,996 @@
+    hmas:isMembershipOf ex:Kate ;
+    hmas:isMembershipFor ex:Director ;
+    hmas:isMembershipIn ex:FL_OperationsDepartment .
+ex:FL_Officer_HumanResources a ex:Officer_HumanResources_Membership ;
+    hmas:isMembershipOf ex:Frank ;
+    hmas:isMembershipFor ex:HumanResourcesOfficer ;
+    hmas:isMembershipIn ex:FL_HumanResourcesDepartment .
+ex:FL_Officer_Finance a ex:Officer_Finance_Membership ;
+    hmas:isMembershipOf ex:Gabriel ;
+    hmas:isMembershipFor ex:FinanceOfficer ;
+    hmas:isMembershipIn ex:FL_FinanceDepartment .
+ex:FL_Planner_Planning a ex:Planner_Planning_Membership ;
+    hmas:isMembershipOf ex:Igor ;
+    hmas:isMembershipFor ex:Planner ;
+    hmas:isMembershipIn ex:FL_PlanningDepartment .
+ex:FL_Carrier_Shipping a ex:Carrier_Shipping_Membership ;
+    hmas:isMembershipOf ex:Jane, ex:Marie, ex:Nancy, ex:Leo ;
+    hmas:isMembershipFor ex:Carrier ;
+    hmas:isMembershipIn ex:FL_ShippingDepartment .
+ex:FL_Collector_Shipping a ex:Collector_Shipping_Membership ;
+    hmas:isMembershipOf ex:Marie, ex:Leo ;
+    hmas:isMembershipFor ex:Collector ;
+    hmas:isMembershipIn ex:FL_ShippingDepartment .
+ex:FL_Deliverer_Shipping a ex:Deliverer_Shipping_Membership ;
+    hmas:isMembershipOf ex:Nancy, ex:Leo ;
+    hmas:isMembershipFor ex:Deliverer ;
+    hmas:isMembershipIn ex:FL_ShippingDepartment .
+
+ex:FL_Planner_Planning ex:canCommunicateWith ex:FL_Carrier_Shipping .
+ex:FL_Carrier_Shipping ex:canCommunicateWith ex:FL_Planner_Planning .
+
+## Process Schemes
+ex:HiringProcess a hmas:ProcessScheme ;
+    hmas:hasRootActivityScheme ex:Hiring .
+ex:HandlingInvoiceProcess a hmas:ProcessScheme .
+ex:ProspectingClientProcess a hmas:ProcessScheme .
+ex:DeliveringServicesProcess a hmas:ProcessScheme .
+
+## Activity Schemes
+ex:Hiring a hmas:ActivityScheme ;
+    ex:isPrecededBy ex:SigningContract ;
+    hmas:achievesOrganizationalGoal ex:Hired .
+ex:SigningContract a hmas:ActivityScheme ;
+    ex:isPrecededBy ex:SelectingCandidate ;
+    hmas:achievesOrganizationalGoal ex:ContractSigned .
+ex:SelectingCandidate a hmas:ActivityScheme ;
+    ex:isPrecededBy ex:AssessingRecommendations, ex:InterviewingCandidates ;
+    ex:isBiasedBy ex:AssessingRecommendations ;
+    hmas:achievesOrganizationalGoal ex:CandidateSelected .
+ex:AssessingRecommendations a hmas:ActivityScheme ;
+    ex:isPrecededBy ex:SelectingInterviewees .
+ex:InterviewingCandidates a hmas:ActivityScheme ;
+    ex:isPrecededBy ex:SelectingInterviewees ;
+    ex:isFacilitatedBy ex:AssessingRecommendations .
+ex:SelectingInterviewees a hmas:ActivityScheme ;
+    ex:isPrecededBy ex:AnnouncingJobPosition ;
+    hmas:achievesOrganizationalGoal ex:IntervieweesSelected .
+ex:AnnouncingJobPosition a hmas:ActivityScheme ;
+    ex:isPrecededBy ex:WritingJobPosition ;
+    hmas:achievesOrganizationalGoal ex:JobPositionAnnounced .
+ex:WritingJobPosition a hmas:ActivityScheme ;
+    hmas:achievesOrganizationalGoal ex:JobPositionWritten .
```

Comment:
I'm leaving this comment to make salient that `hmas:ProcessScheme` and `hmas:ActivityScheme` are used here. If the scenario that introduces these terms applies changes to the form or utility of the terms, then this part will need to be updated.